_data/import-public.rb

```diff
@@ -17,3 +19,17 @@
     end
   end
 end
+
+data = JSON.load File.read(File.join(DATA_DIR, 'team.json'))
+fm_rexexp = Regexp.new('^---.*---$', Regexp::MULTILINE)
+
+Dir[File.join [DATA_DIR] + %w(.. _team *.md)].each do |member_file|
+  name = File.basename(member_file)[0..-4] # Trim .md extension
```

Comment:
@mbland is it worth splitting on `.` instead of trimming the string? Jekyll supports other file extensions like `.markdown` or `.textile`